dialects/affine/passes/lower_for.h

```diff
@@ -9,11 +9,22 @@ namespace thorin::affine {
 class LowerFor : public RWPass<LowerFor, Lam> {
 public:
     LowerFor(PassMan& man)
-        : RWPass(man, "lower_affine_for") {}
+        : RWPass(man, "lower_affine_for")
```

Comment:
We can leave this in as an example.
But as you write above, the speedup is not worth the effort, indirection, redundancy, and maintaining overhead.
(This comment is with regard to the sym_ construction)

Reply:
yeah, I think I will just remove this again